fhir-examples/src/main/resources/json/ibm/complete-absent/Account-1.json

```diff
@@ -1,13 +1,5 @@
 {
     "resourceType": "Account",
-    "_id": {
-        "extension": [
-            {
-                "url": "http://hl7.org/fhir/StructureDefinition/data-absent-reason",
```

Comment:
I take it this is no longer used? it's absent = therefore it should be treated as absent?

Reply:
Crazy question, what happens when you leave it?  If it'd be valid, maybe we should have a test with data like this?

Reply:
as of 4.0.1 that is invalid.  i did manually test this and it complains its an unrecognized element (which i think is the right behavior), but didn't introduce a negative test case for it.